include/swupd.h

```diff
@@ -45,17 +50,20 @@ struct sub {
 
 	/* oldversion: set to 0 by calloc(), possibly overridden by MoM read */
 	int oldversion;
+	int is_mix : 1;
```

Comment:
this new field for `struct sub` does not appear to be used, so you can remove it